tasks/Receptionist.tex

```diff
@@ -23,36 +24,45 @@ \subsection*{Setup}
-	during setup period before competition (if not using door knocking).
+	\item \textbf{Arriving Guests:} Both guests have a name, favorite drink, and a interest. An arriving guest will either step in front of the robot or wait behind the door to step in if the door is closed. Guests have to be guided to the beverage area and then the living room where the robot will introduce the guests to each other. Each of the guest will arrive separately. 
+	
+	\item \textbf{Passive Guests:} Other Guests: A few individuals, including possibly spectators, are standing in small groups of two or three, all over the arena but not covering relevant places e.g. the couch, seats or the area with the drinks.
 	\end{itemize}
 
 \end{itemize}
 
 \subsection*{Procedure}
+Both guests arrive separately. The robot either opens the door for the guest or waits for them at the starting point. It greets the guest and asks for their name. The robot then guides the guest to the beverage area, where it asks for their favorite drink and checks if the drink is available on the table and where it stands. The robot finds out the interest of the guest at a freely chosen moment. After showing the guest the beverage area, the robot escorts them to the living room and offers a free seat. Once both guests are seated, the robot introduces them to each other.
 \begin{itemize}
-    \item \textbf{Introductions:} When introducing guests, the robot must clearly identify the person being introduced and state their name and favorite drink. Introducing two people means to introduce them to each other.
+	\item \textbf{Greeting guests:} the person paces a little to the left and right during conversation with the robot. other people might appear in the background
+	
+	\item \textbf{Looking at person:} During verbal interactions and descriptions of people, robot 
+	looks at the conversational partner. The conversational partner will make small movements to each side to confirm the robot is dynamically looking at the person. Points for looking at the person talking, will only be awarded if the robot proves to continuously look at the moving person. 
+	
+	\item \textbf{Looking at direction of navigation:} During navigation robot looks in the direction where it is going. Persistently gazing towards unrelated person or incorrect direction while moving during the task deducts points. 
+    
+    \item \textbf{Smalltalk} ask each guest for one interest 
+    
+    \item \textbf{Finding the drink} the robot shows both guest the drinking area where it will ask for their favorite drink and tell the guest if and where (left, center right is enough) that drink is available on the table. 
```

Comment:
```suggestion
    \item \textbf{Finding the drink} The robot shows both guest the drinking area where it will ask for their favorite drink and tell the guest if and where (left, center right is enough) that drink is available on the table. 
```